src/resources/elements/EtherscanLink/EtherscanLink.html

```diff
@@ -4,13 +4,13 @@
        href.to-view="networkExplorerUri" if.to-view="networkExplorerUri"
        ptooltip.bind="address"
        rel="noopener noreferrer" target='${internal ? "" : "_blank"}'>
-      ${text || address}
+      ${ens || text || address}
```

Comment:
```suggestion
      ${text || ens || address}
```

because we want `text` to overwrite the others?